source/includes/steps-create-system-dsn.yaml

```diff
@@ -104,14 +104,15 @@ content: |
          another ODBC driver, fill in this field with the
          string ``mongosql_auth``.
 ---
-title: (Optional) If SSL is enabled, fill in the SSL form fields. 
+title: (Optional) If TLS/SSL is enabled, fill in the TLS/SSL form fields. 
 ref: form-fields-ssl
 level: 4
 stepnum: 7
 content: |
-  Access the SSL form fields by clicking :guilabel:`Details >>`, then
-  the :guilabel:`SSL` tab. The following parameters are required
-  when :ref:`TLS/SSL <msqld-mongodb-tls-options>` is enabled:
+  Access the TLS/SSL form fields by clicking :guilabel:`Details >>`,
+  then the :guilabel:`SSL` tab. Fill in the following parameters
+  depending on your :ref:`TLS/SSL <msqld-mongodb-tls-options>`
+  configuration:
```

Comment:
It's too bad we can't give opinionated guidance here, but SSL configurations are so situation-specific I don't see how we can. Maybe say something like "The following parameters are available and may be required, depending on your ref:`TLS/SSL <msqld-mongodb-tls-options>` configuration:" Or just leave it as is.